### PR TITLE
Simplify EBL absorption spectral model

### PR DESCRIPTION
**Pull request by QRemy**
This PR simplify EBL absorption spectral model as proposed in PIG 21: #2944

### CODE REVIEW

File: gammapy/modeling/models/spectral.py [hunk 1]
```diff
@@ -1451,18 +1450,22 @@ class Absorption:
...
+    alpha_norm: float
+        Norm of the EBL model
     interp_kwargs : dict
         Interpolation option passed to `ScaledRegularGridInterpolator`.
         By default the models are extrapolated outside the range. To prevent
         this and raise an error instead use interp_kwargs = {"extrapolate": False}
     """
 
-    tag = "Absorption"
+    tag = ["EBLAbsorptionNormSpectralModel", "ebl-absorption"]
```
Comment (adonath) [inline gammapy/modeling/models/spectral.py:1463]: Include "norm" in the short tag here? Like `ebl-norm` or `ebl-absorbtion-norm`?

File: gammapy/modeling/models/spectral.py [hunk 2]
```diff
@@ -1451,18 +1450,22 @@ class Absorption:
...
+    alpha_norm = Parameter("alpha_norm", 1.0, frozen=True)
+    redshift = Parameter("redshift", 0.1, frozen=True)
 
-    def __init__(self, energy, param, data, filename=None, interp_kwargs=None):
-        self.filename = filename
+    def __init__(self, energy, param, data, redshift, alpha_norm, interp_kwargs=None):
+        self.filename = None
         # set values log centers
         self.param = param
```
Comment (adonath) [inline gammapy/modeling/models/spectral.py:1470]: Is the `.param` still used? I think it's fixed to `redshift` now...

File: gammapy/modeling/models/spectral.py [hunk 3]
```diff
@@ -1572,137 +1596,20 @@ def read_builtin(cls, name, interp_kwargs=None):
...
-
         """
         models = dict()
         models["franceschini"] = "$GAMMAPY_DATA/ebl/ebl_franceschini.fits.gz"
         models["dominguez"] = "$GAMMAPY_DATA/ebl/ebl_dominguez11.fits.gz"
         models["finke"] = "$GAMMAPY_DATA/ebl/frd_abs.fits.gz"
 
-        return cls.read(models[name], interp_kwargs=interp_kwargs)
-
-    def table_model(self, parameter):
```
Comment (adonath) [inline gammapy/modeling/models/spectral.py:1588]: I guess one could convert it to a `PiecewiseNormSpectralModel`, but I'm not sure if there is any real use-case for this, so removing is fine I guess...

**Review by adonath — APPROVED** [review]
Thanks a lot @QRemy, I have left two minor inline comments...

File: gammapy/modeling/models/spectral.py (hunk 2 above)
Comment (QRemy) [reply gammapy/modeling/models/spectral.py:1470]: I think we still need it to setup the ScaledRegularGridInterpolator and interpolate the table at the redshift value.

File: gammapy/modeling/models/spectral.py (hunk 3 above)
Comment (QRemy) [reply gammapy/modeling/models/spectral.py:1588]: It was used only in the test_absorbed_extrapolate, I just replace it using evaluate
https://github.com/gammapy/gammapy/blob/ba65d5aa7be4bd77f4778c48420db499dd8d28d8/gammapy/modeling/models/tests/test_spectral.py#L476

File: gammapy/modeling/models/spectral.py (hunk 2 above)
Comment (adonath) [reply gammapy/modeling/models/spectral.py:1470]: Indeed you are right...

**Comment by adonath** [comment]
Thanks @QRemy!

**Pull request merged**